open-rosa/src/main/java/org/odk/collect/openrosa/forms/OpenRosaClient.kt

```diff
@@ -0,0 +1,152 @@
+package org.odk.collect.openrosa.forms
+
+import org.odk.collect.forms.FormListItem
+import org.odk.collect.forms.FormSource
+import org.odk.collect.forms.FormSourceException
+import org.odk.collect.forms.FormSourceException.AuthRequired
+import org.odk.collect.forms.FormSourceException.FetchError
+import org.odk.collect.forms.FormSourceException.SecurityError
+import org.odk.collect.forms.FormSourceException.ServerNotOpenRosaError
+import org.odk.collect.forms.ManifestFile
+import org.odk.collect.openrosa.forms.OpenRosaXmlFetcher.WebCredentialsProvider
+import org.odk.collect.openrosa.http.OpenRosaConstants
+import org.odk.collect.openrosa.http.OpenRosaHttpInterface
+import org.odk.collect.openrosa.parse.OpenRosaResponseParser
+import java.io.InputStream
+import java.net.HttpURLConnection
+import java.net.UnknownHostException
+import java.util.concurrent.Callable
+import javax.net.ssl.SSLException
+
+class OpenRosaClient(
```

Comment:
I renamed this and did the conversion to Kotlin as it's really the main component exposed from `open-rosa` now, and I was going to end up making modifications to add integrity URL support.